scripts/HoveredMidiRelative/HoveredMidiRelativeExt.py

```
@@ -516,119 +626,164 @@ def onReceiveMidiSlotLearn(self, index: int):
-		
-		self.display_manager.update_parameter_display(self.activePar)
+		try:
+			# Handle ParGroup
+			if ParameterValidator.is_pargroup(self.activePar):
+				self.undo_manager.create_reset_undo_for_pargroup(self.activePar)
+			else:
+				# Handle single Par
+				self.undo_manager.create_reset_undo_for_parameter(self.activePar)
+			
+			self.display_manager.update_parameter_display(self.activePar)
+		except:
```

Ruff (0.14.3)

Do not use bare `except`

(E722)